app/views/shared/views/ems_common/angular/_form.html.haml

```diff
@@ -261,7 +261,7 @@
                             :change_stored_password => _("Change stored client key"),
                             :cancel_password_change => _("Cancel client key change"),
                             :verify_title_off       => _("Tenant ID, Client ID and matching Client Key fields are needed to perform verification of credentials")}
-    %div{"ng-if" => "emsCommonModel.emstype != '' && emsCommonModel.emstype != 'azure'"}
+    %div{"ng-show" => "emsCommonModel.emstype != '' && emsCommonModel.emstype != 'azure'"}
```

Comment:
The downside to `ng-show` is that all the fields are there, always.

You can't have multiple ng-show statemens with different conditions including the same partial because then, the partial will always exist multiple times.

(Unless that partial supports some kind of prefix to regulate this.)

Reply:
((If you really need the partial to exist and not be visible, use `ng-if` with a .hidden class, but ... better fix the underlying problem really.))

Reply:
In this specific case... this looks like `isBasicInfoValid` is already doing logic to determine which fields should be there.

So I'm almost sure any fix for your issue should be to `isBasicInfoValid` itself, not the partial.